framework/src/postprocessors/FunctionValuePostprocessor.C

```diff
@@ -18,14 +18,22 @@ FunctionValuePostprocessor::validParams()
   InputParameters params = GeneralPostprocessor::validParams();
   params.addRequiredParam<FunctionName>("function",
                                         "The function which supplies the postprocessor value.");
+
   params.addParam<Point>(
       "point", Point(), "A point in space to be given to the function Default: (0, 0, 0)");
+  params.addParam<std::vector<PostprocessorName>>(
+      "point_postprocessors",
+      "A tuple of Postprocessors that are passed to the function as the space "
+      "argument. Cannot be specified if point argument is specified.");
```

Comment:
Couldn't this be one parameter? I think there is default value coupling for postprocessors.

Reply:
I thought that would work, but it doesn't...or I did it wrong.

Reply:
and `coupledPostprocessors("point")` to get the number of vector elements

Reply:
If you change the value `y_pp` in param `point_postprocessors` in file `pps_args_function_value_pps.i` to 0, it should give the same result as before. But it fails like this:

```
postprocessors/function_value_pps.pps_args_function_value: *** ERROR ***
postprocessors/function_value_pps.pps_args_function_value: The following Reporter(s) were not declared:
postprocessors/function_value_pps.pps_args_function_value: 
postprocessors/function_value_pps.pps_args_function_value: Postprocessor "0":
postprocessors/function_value_pps.pps_args_function_value:   Producer:
postprocessors/function_value_pps.pps_args_function_value:     None
postprocessors/function_value_pps.pps_args_function_value:   Consumer(s):
postprocessors/function_value_pps.pps_args_function_value:     FunctionValuePostprocessor "val" (mode: ROOT)
```

The handling of default values of PPs does not work for vectors of PPNames? It was similar for coupled vars before some significant effort a few years ago, wasn't it?

Reply:
That's because you;re using the wrong API. Don;t use `getPostprocessorValueByName` use `getPostprocessorValue`.

Reply:
Unfortunately, the controllable point parameter is used in the controls test as primary way of testing controlling parameters of different types. I replaced those tests with a test object.